src/data/linux-quiz.ts

```diff
@@ -1330,7 +1330,70 @@ const linuxQuiz = [
     Distractor3: "stop [signal] PID",
     Explanation: "The kill command is used to stop a running process.",
     Link: "https://www.linuxfoundation.org/blog/blog/classic-sysadmin-how-to-kill-a-process-from-the-command-line"
-  }
+  },
+  {
+   Question: "In Linux, what command is used to create a RAM disk or block device with specified size and file system?",
+   Answer: "dd",
+   Distractor1: "mkfs",
+   Distractor2: "mount",
+   Distractor3: "fdisk",
+   Explanation: "The 'dd' command can be used to create a RAM disk by copying data with a specified size, which effectively creates a block device with the desired size and can be formatted with a file system afterward.",
+   Link: "https://www.linuxjournal.com/content/creating-ram-disk-device"
+   },
+   {
+      Question: "In Linux, which command is used to perform a binary file comparison and display the differences between two files?",
+      Answer: "cmp",
+      Distractor1: "diff",
+      Distractor2: "compare",
+      Distractor3: "contrast",
+      Explanation: "The 'cmp' command is used to compare two binary files byte by byte and displays the differences between them, if any.",
+      Link: "https://linux.die.net/man/1/cmp"
+    },
+    {
+      Question: "What Linux command is used to search for text patterns in files and supports regular expressions for advanced pattern matching?",
+      Answer: "grep",
+      Distractor1: "find",
+      Distractor2: "search",
+      Distractor3: "locate",
+      Explanation: "The 'grep' command is used to search for text patterns in files and supports powerful regular expressions for advanced pattern matching.",
+      Link: "https://linux.die.net/man/1/grep"
+    },
```

Comment:
we already have a couple of similar questions for the grep command in this file
let's remove this

```suggestion
```